modules/testsuite/cxf-tests/src/test/resources/jaxws/cxf/http2/WEB-INF/web.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<web-app
+        version="3.0" xmlns="http://java.sun.com/xml/ns/javaee"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd">
```

Comment:
This is to be hyper-correct, obviously the original content would work too, but shouldn't default to use outdated examples.
I know it is used across the project a lot like this as it wasn't migrated to EE10 yet, but we could start somewhere (here):

```suggestion
<web-app
        version="6.0" xmlns="https://jakarta.ee/xml/ns/jakartaee"
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/web-app_6_0.xsd">
```

Reply:
I think we only need to change some of them and make them co-exist. We still have the thing to check the legacy web.xml and other deployment descriptor still work. WDYT ?

Reply:
I don't see why we should keep outdated versions.

Reply:
I saw there are still old version web.xml files(java.sun.com namespace and version 2.x or 3.0) in WFLY testsuite:https://github.com/wildfly/wildfly/blob/main/testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/web/web.xml.  Is it for this purpose ?

Reply:
I doubt it is on purpose, it just wasn't updated when whole thing was converted to EE10.